src/vs/workbench/parts/terminal/electron-browser/media/terminal.css

```diff
@@ -29,6 +29,7 @@
 	position: absolute;
 	bottom: 0;
 	top: 0;
+	padding-bottom: 2px;
```

Comment:
Does `bottom: 2px` here save us from doing the changes in `terminalInstance.ts`?